packages/Microsoft.SqlToolsService/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Microsoft.SqlToolsService
+{
+    public class Class1
```

Comment:
do you know the purpose of this file?

Reply:
Yes, dotnet pack requires a project to actually build and produce output even when it's not used.